src/helpers/bestSoldTv.js

```diff
@@ -0,0 +1,10 @@
+import {bestSellingTv} from "../constants/inventory.js";
+
+
+function bestSoldTv() {
+    return bestSellingTv.brand + " " + bestSellingTv.type + " - " + bestSellingTv.name;
```

Comment:
Ik vind het volgende altijd wat beter te lezen
```suggestion
    return `${bestSellingTv.brand} ${bestSellingTv.type} - ${bestSellingTv.name}`;
```